CONTRIBUTING.md

```diff
@@ -54,6 +58,16 @@ python3 -m scripts.generate_type_stubs_and_docs
 # Documentation will be dumped into ../docs/python/
 ```
 
+#### Java
+To update the javadocs for the java bindings, you can simply run:
+
+```shell
+cd java
+mvn package
```

Comment:
I generated these manually using `mvn package` -- is there a better way to generate these javadocs?  `mvn javadoc:javadoc` runs, but doesn't respect the `outputDirectory` configuration parameter in the POM.